src/Accordion/readme.md

```diff
@@ -17,49 +17,70 @@ Type: Form Component
+```jsx
+/** If you use hash router, you need to pass the hash in the href */
+const accordionList: Array<AccordionItemProps> = [
+    { header: "First", children: "Some content" },
+    { header: "Second", subHeader: "Some description", children: <><h4>This is the second</h4><p>Some rich content</p></> },
+    { header: "Third", children: <SomeComponent>content</SomeComponent> },
+];
+function onToggle(e: React.MouseEvent<HTMLButtonElement>) {
+    /** Run any necessary logic before setting the defaultValue */
+    setValue(Number(e.currentTarget.dataset.indexNumber));
+}
+```
 
-| Property            | Type                                   | Description                                                                            |
-|---------------------|----------------------------------------|----------------------------------------------------------------------------------------|
-| alternative?        | `boolean`                              | Toggle alternative style of accordion                                                  |
-| className?          | `string`                               | Element class                                                                          |
-| activeIndex?        | `number`                               | index of the default active item                                                       |
-| customIcon?         | `JSX.Element`                          | Custom icon for the accordion trigger                                                  |
-| customIconExpanded? | `JSX.Element`                          | Custom icon to be used when expanded. This will add a transition between the two icons |
-| iconPosition?       | `string`                               | Accordion icon placement<sup>1</sup>                                                   |
-| iconTransition?     | `string`                               | Icon transition rotation degree<sup>2</sup>                                            |
-| id?                 | `string`                               | Element id                                                                             |
-| list                | `Array<AccrodionListItem>`<sup>3</sup> | List of accordion items                                                                |
+## Properties
 
-## Footnote
+#### AccordionProps
+This interface extends all native attributes of `HTMLDivElement`, adding the following extra attributes:
 
-1. Icon positions supported are: `right` and `left`
-2. Icon transitions supported are: `"deg-180"`, `"deg-180-counter"`, `"deg-90"`, `"deg-90-counter"`
-3. Property `list` has an exported interface named `AccordionListItem`:
+| Property     | Type                                         | Description                                                    |
+| ------------ | -------------------------------------------- | -------------------------------------------------------------- |
```

Comment:
you should mention defaultValue here since it's actually being used for something

Reply:
Even though `defaultValue` is a native attribute, I agree with the fact that I need to point out that it is used to define the value of the breadcrumb if needed